app/graphql/types/query_type.rb

```diff
@@ -71,6 +75,14 @@ def weather_map(id: nil)
       WeatherMapService.new.import
     end
 
+    def survey_comments(survey_id: nil)
```

Comment:
is `survey_comments(survey_id: nil)` nicht identisch zu `survey_comments(survey_id)`, oder was bewirkt der fallbackwert `nil`?

Reply:
Mit _survey_id: nil_ ist survey_id optional, andernfalls ein Pflichtwert